editor/js/libs/ui.js

```diff
@@ -534,6 +534,25 @@ UI.Color.prototype.setHexValue = function ( hex ) {
 
 // Number
 
+UI.nudgeNumericValue = function ( originalValue, keyCode, shiftKeyDown ) {
+	var parsedValue = -1;
+	try {
+		parsedValue = parseFloat( originalValue );
+	} catch ( e ) {
+		return originalValue;
+	}
+
+	var direction = 0;
+	if (keyCode === 38) {
```

Comment:
Can you please improved the code style a little bit, see https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2

There is also an online tool: http://zz85.github.io/mrdoobapproves/